lib/recurly/base.php

```diff
@@ -366,7 +364,7 @@ protected static function __parseXmlToObject($node, &$object)
             continue;
           }
 
-          $new_obj = Recurly_Resource::__createNodeObject($node);
+          $new_obj = Recurly_Resource::__createNodeObject($node, $object->_client);
```

Comment:
One reason the tests are failing right now is because in this context, `$object` can be an array (see line 360) which wouldn't have a `_client` object.

Reply:
Thank you! I addressed the issue and the tests are now passing.